src/main/java/frc/robot/Path.java

```diff
@@ -10,10 +10,13 @@
     private ArrayList rightString = new ArrayList<>();
     private ArrayList leftString = new ArrayList<>();
     private static final Logger LOGGER = Logger.getLogger(Robot.class.getName());
-    public Path(){}    
+    private String pathName;
+    public Path(String PathName){
+        pathName = PathName;
+    }    
 
     public ArrayList returnLeftList() throws IOException{
-            File file = new File("/paths/StraightTen.left.pf1.csv");
+            File file = new File(pathName + "left.pf1.csv");
             Scanner sc = new Scanner(file);
             String[] testArray;
```

Comment:
Consider combining returnLeftList and returnRightList into one method that returns a 2D ArrayList (i.e. ArrayList<ArrayList<String>>), or, even better, an array of ArrayLists